Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -155,6 +155,155 @@ static int CompareGameListItems(const GameListItem* iso1, const GameListItem* is
   return 0;
 }
 
+static std::unordered_map<std::string, std::string> LoadCustomTitles()
+{
+  // Load custom game titles from titles.txt
+  // http://www.gametdb.com/Wii/Downloads
+  const std::string& load_directory = File::GetUserPath(D_LOAD_IDX);
+
+  std::ifstream titlestxt;
+  OpenFStream(titlestxt, load_directory + "titles.txt", std::ios::in);
+
+  if (!titlestxt.is_open())
+    OpenFStream(titlestxt, load_directory + "wiitdb.txt", std::ios::in);
+
+  if (!titlestxt.is_open())
+    return {};
+
+  std::unordered_map<std::string, std::string> custom_titles;
+
+  std::string line;
+  while (!titlestxt.eof() && std::getline(titlestxt, line))
+  {
+    const size_t equals_index = line.find('=');
+    if (equals_index != std::string::npos)
+    {
+      custom_titles.emplace(StripSpaces(line.substr(0, equals_index)),
+                            StripSpaces(line.substr(equals_index + 1)));
+    }
+  }
+
+  return custom_titles;
+}
+
+static std::vector<std::string> GetFileSearchExtensions()
+{
+  std::vector<std::string> extensions;
+
+  if (SConfig::GetInstance().m_ListGC)
+  {
+    extensions.push_back(".gcm");
+    extensions.push_back(".tgc");
+  }
+
+  if (SConfig::GetInstance().m_ListWii || SConfig::GetInstance().m_ListGC)
+  {
+    extensions.push_back(".iso");
+    extensions.push_back(".ciso");
+    extensions.push_back(".gcz");
+    extensions.push_back(".wbfs");
+  }
+
+  if (SConfig::GetInstance().m_ListWad)
+    extensions.push_back(".wad");
+
+  if (SConfig::GetInstance().m_ListElfDol)
+  {
+    extensions.push_back(".dol");
+    extensions.push_back(".elf");
+  }
+
+  return extensions;
+}
+
+static bool ShouldDisplayGameListItem(const GameListItem& item)
+{
+  bool list = true;
+
+  switch (item.GetPlatform())
+  {
+  case DiscIO::Platform::WII_DISC:
+    if (!SConfig::GetInstance().m_ListWii)
```

Comment:
It was probably just because nobody thought of splitting it off to separate functions (or lambda expressions) so that that would be possible. Feel free to do the change.